src/includer/parsers.ts

```diff
@@ -110,6 +110,15 @@ function tagsFromSpec(spec: OpenAPISpec): Map<string, Tag> {
         }
     }
 
+    parsed.forEach((tag: Tag) => {
+        if (tag['x-navtitle']) {
+            tag.name = tag['x-navtitle'];
+        }
+        if (tag['x-slug']) {
+            tag.id = tag['x-slug'];
```

Comment:
Since we're now modifying `tag.id` based on `x-slug`'s value, I would argue this would actually break a (loose) contract on `Map<string, Tag>` we had prior to these changes, i.e. a `Tag`'s name now may or may not correspond to the key under which it is stored in the map.

From my point of view, this fact makes using a map obsolete, and we should probably collect the tags in a list instead.

Reply:
That's why I had to modify `includer/index.ts`.

But we could see this as `Map<OpenAPI_tag_name, diplodoc_section>`, we still need to collect all tags from endpoints in OpenAPI spec file.